### ISSUE TITLE
Adding a new sprite to a parent with cascadeColorEnabled doesn't apply color

### ISSUE THREAD

**Issue by nheagy** [cocos2d/cocos2d-objc#650]
I expect a new node to inherit its parent color if the parent has cascadeColorEnabled, but it currently appears with the default color (white).

**Comment by slembcke** [comment]
This is not a feature that I've ever used (It's odd so I've avoided it). If I remember correctly, the way it works is that the child node's cascadeColorEnabled value is what control if it inherits from it's parent and not the other way around. It's pretty old functionality, and we can't change it now.

**Comment by nheagy** [comment]
You don't use the feature, you misremember how it works, and yet you've closed the issue…

**Comment by slembcke** [comment]
You are correct. :-\ Sorry about that. I replied to that at 2:00 AM while waiting at an airport.

There are tests for the cascadeColorEnabled property here: https://github.com/cocos2d/cocos2d-iphone/blob/develop-v3/cocos2d-ui-tests/tests/ColorTest.m Do you know what you are doing differently from them that isn't working?
